src/webui/www/public/filters.html

```diff
@@ -7,4 +8,8 @@
                     <li id="paused_filter"><a href="#" onclick="setFilter('paused');return false;"><img src="images/skin/paused.png"/>QBT_TR(Paused)QBT_TR</a></li>
                     <li id="active_filter"><a href="#" onclick="setFilter('active');return false;"><img src="images/skin/filteractive.png"/>QBT_TR(Active)QBT_TR</a></li>
                     <li id="inactive_filter"><a href="#" onclick="setFilter('inactive');return false;"><img src="images/skin/filterinactive.png"/>QBT_TR(Inactive)QBT_TR</a></li>
-                </ul>
\ No newline at end of file
+                </ul>
+                <br/>
+                QBT_TR(Labels)QBT_TR
+                <ul id="filterLabelList" style="padding-left: 0px;">
```

Comment:
And since `filterList` has `padding-left: 0px;`, you can remove the `style` attribute.